baiji/config.py

```diff
@@ -76,6 +94,13 @@ def key(self):
     def secret(self):
         return self._try(['AWS_SECRET_ACCESS_KEY', 'AWS_SECRET'], 'AWS_SECRET')
 
+    @property
+    def region(self):
+        return self._try(['AWS_DEFAULT_REGION'], 'REGION')
+
+# Alias to avoid breaking changes
+Credentials = Settings
```

Comment:
Ah, similar discussion above. I'll leave it up to you. If you do decide to keep it as is, it should really only be used by external users (internally we should use `settings` throughout). Also, it should be clear that it should be removed in the next breaking change.